config/src/main/java/tech/pegasys/pantheon/config/IbftConfigOptions.java

```diff
@@ -37,6 +37,6 @@ public int getBlockPeriodSeconds() {
   }
 
   public int getRequestTimeoutMillis() {
-    return ibftConfigRoot.getInteger("requesttimeout", DEFAULT_ROUND_EXPIRY_MILLISECONDS);
+    return ibftConfigRoot.getInteger("requesttimeoutseconds", DEFAULT_ROUND_EXPIRY_MILLISECONDS);
```

Comment:
shouldn't this be requesttimeoutmilliseconds the default value and method suggests it is meant to represents millis